src/uniparc/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,291 @@
+          </p>
+          <p>
+            <Link
+              to={{
+                pathname: LocationToPath[Location.UniParcResults],
+                search: stringifyQuery({ query: '*' }),
+              }}
+            >
+              Start searching in UniParc »
+            </Link>
+          </p>
+        </div>
+
+        {/* Table */}
+        <section
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            `uniprot-grid-cell--medium-span-${expanded ? '12' : '4'}`
+          )}
+        >
+          <h2>Top Databases</h2>
+          <div className={styles['table-container']}>
+            {groupedDatabases.map((databases, index) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <table key={index}>
+                <thead>
+                  <tr>
+                    <th>Cross-reference</th>
+                    <th>
+                      Number of
+                      <br />
+                      UniParc entries
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {databases.map(({ count, value, label }) => (
+                    <tr key={value}>
+                      <td>
+                        <Link
+                          to={{
+                            pathname: LocationToPath[Location.UniParcResults],
+                            search: stringifyQuery({
+                              query: '*',
+                              facets: `${FacetsEnum.Database}:${value}`,
+                            }),
+                          }}
+                        >
+                          {label || value}
+                        </Link>
+                      </td>
+                      <td className={styles.end}>
+                        <LongNumber>{count}</LongNumber>
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+            ))}
+          </div>
+
+          <Button
+            variant="tertiary"
+            onClick={() => setExpanded((expanded) => !expanded)}
+          >
+            {expanded ? 'View less' : 'Explore more'} databases
+          </Button>
+        </section>
+
+        {/* Text */}
+        <section
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            `uniprot-grid-cell--medium-span-${expanded ? '9' : '5'}`
+          )}
+        >
+          <h2>Searching the Database</h2>
+          <br />
+          <p>
+            UniParc is available for text- and sequence-based searches.
+            Performing a similarity search against UniParc is equivalent to
+            performing the same search against all databases cross-referenced in
+            UniParc, as UniParc contains all proteins from its source databases.
+          </p>
+          <p>
+            Sequence similarity searches can be done using{' '}
```

Comment:
Just an idea here. The landing page seems like a place to list activities the user can try. The `»` in "Start searching in UniParc »" makes me think "go off and try this out". What about here we change the wording a bit:
```
Start a sequence similarity search in UniParc using BLAST »
```